client/app/js/controllers/main.js

```diff
@@ -549,7 +524,7 @@ controller('DisableEncryptionCtrl', ['$scope', '$uibModalInstance', function($sc
       $uibModalInstance.close(true);
     };
 }]).
-controller('IntroCtrl', ['$scope', '$rootScope', '$uibModalInstance', function ($scope, $rootScope, $uibModalInstance) {
+controller('IntroCtrl', ['$scope', '$rootScope', '$uibModalInstance', 'GLTranslate', function ($scope, $rootScope, $uibModalInstance, GLTranslate) {
```

Comment:
@evilaliv3 I believe the IntroCtrl is not used anywhere in the code. Can we delete it?